### ISSUE TITLE
Kotlin build issue after 7.24.0 update

### ISSUE THREAD

**Issue by LaVielle** [wix/react-native-navigation#7385]
## 📚 Documentation

Android build was failing after updating from RNN `^7.23.1-hotfix.1` to `^7.24.0`, with an error message containing:

```
The binary version of its metadata is 1.5.1, expected version is 1.1.16
```

After looking into [this SO post](https://stackoverflow.com/questions/67448034/module-was-compiled-with-an-incompatible-version-of-kotlin-the-binary-version) I figured the Kotling version suggested in the RNN docs is too old.

Updating to the [latest version of Kotlin](https://kotlinlang.org/docs/releases.html#release-details) solved the build issue.

If you don't see any downside to update to Kotlin 1.6.0, maybe you could update the Installation Docs, and mention the upgrade requirements in the 7.24.0 release notes.

Anyways, thanks for the update. I appreciate the quick fix of the status bar offset!

**Comment by swabbass** [comment]
The suggested Kotlin version in docs might be outdated, but the actual version in the code is `1.5.31`, [see](https://github.com/wix/react-native-navigation/blob/master/lib/android/app/build.gradle#L18) , we might need to update the docs.

**Comment by yogevbd** [comment]
Closed via #7391
